lxd/patches.go

```diff
@@ -2247,6 +2248,63 @@ func patchStorageApiInsertZfsDriver(name string, d *Daemon) error {
 	return nil
 }
 
+func patchStorageZFSnoauto(name string, d *Daemon) error {
+	pools, err := dbStoragePools(d.db)
+	if err != nil {
+		if err == NoSuchObjectError {
+			return nil
+		}
+		logger.Errorf("Failed to query database: %s", err)
+		return err
+	}
+
+	for _, poolName := range pools {
+		_, pool, err := dbStoragePoolGet(d.db, poolName)
+		if err != nil {
+			logger.Errorf("Failed to query database: %s", err)
+			return err
+		}
+
+		if pool.Driver != "zfs" {
+			continue
+		}
+
+		zpool := pool.Config["zfs.pool_name"]
+		if zpool == "" {
+			continue
+		}
+		path := fmt.Sprintf("%s/containers", zpool)
+
+		output, err := shared.RunCommand(
```

Comment:
Hm, why are you listing this manually and not go through the db by requesting all defined storage volumes of type container?